### ISSUE TITLE
Ensure redaction text does not go off of page

### ISSUE THREAD

**Issue by gravitystorm** [openstreetmap/openstreetmap-website#1845]
The redaction text at https://www.openstreetmap.org/redactions/101 doesn't wrap, and so it makes the page super-wide. I don't think there's any reason to avoid wrapping here, even in pre-formatted blocks.

If there is some reason to prevent wrapping, then the block should be scrollable but the page width otherwise unaffected.

**Comment by robotscissors** [comment]
It looks like you applied some CSS and popped the code into a container. I figured that was going to be my approach. If you agree, I will take a look at the other markdown pages as you suggested.

**Comment by pnorman** [comment]
I didn't change any CSS, I just edited the text of the message.

**Comment by robotscissors** [comment]
Understood. Let me look at your solution and see if it needs to be applied elsewhere.

**Comment by pnorman** [comment]
I didn't have a solution, I edited the markdown of the redaction. No code was changed. Other redactions with a long line in a markdown code block would have the same problem.

**Comment by robotscissors** [comment]
@pnorman HI Paul, I am a newbie and this is a huge application, so forgive me.  

I am having trouble recreating the issue in my current environment. He is what I have done.

- I installed the application locally and was able to see the website at localhost:3000. 
- I created an account for myself and validated it through an email link that was displayed on the server.
- I created a diary entry with a long string using markdown `` to encapsulate the entire message for the worst case, but it wrapped just fine.

Any other hints to recreate the issue in the development environment.
